.scalafmt.conf

```diff
@@ -2,15 +2,14 @@ version = 2.0.0-RC5
 
 style = defaultWithAlign
 
-danglingParentheses        = true
 docstrings                 = JavaDoc
 indentOperator             = spray
 maxColumn                  = 120
 rewrite.rules              = [RedundantParens, SortImports, AvoidInfix]
 unindentTopLevelOperators  = true
 align.tokens               = [{code = "=>", owner = "Case"}]
-align.openParenDefnSite    = true
-align.openParenCallSite    = true
+align.openParenDefnSite    = false
```

Comment:
Redundant? Default changed from true to false in v1.6. https://scalameta.org/scalafmt/docs/configuration.html#alignopenparendefnsite